sdk/python/test_data/pipelines/xgboost_sample_pipeline.yaml

```diff
@@ -132,19 +132,19 @@ components:
           defaultValue: gbtree
           parameterType: STRING
         label_column:
-          defaultValue: '0'
+          defaultValue: 0.0
```

Comment:
Note that these changes were not the intend of the bug, but the improved `default_value` setting logic caught this type inconsistency that was previously bypassed.